pkg/report-emailer/report_emailer.go

```diff
@@ -155,7 +155,9 @@ func (re *ReportEmailer) CreateReport(schedule datasource.Schedule, authConfig *
 			bugsnag.Notify(err)
 		} else {
 			log.DefaultLogger.Debug(fmt.Sprintf("ReportEmailer.createReport: schedule: %s", schedule.Name))
-			report := reporter.CreateNewReport(scheduleID, schedule.Name)
+			schedularFileName := GetFormattedFileName(schedule.Name, schedule.DateFormat, schedule.DatePosition)
+			log.DefaultLogger.Info("schedularFileName", schedularFileName)
```

Comment:
@anildahalsussol remove the repeating log entries of `schedularFileName` and only keep one log type once please. Otherwise the same data will show multiple times in console. Maybe do it for other logs too.